functions/index.js

```diff
@@ -469,6 +469,44 @@ module.exports.reloadsheetdata = functions.https.onRequest(async (req, res) => {
   res.status(200).send(html_snippets);
 });
 
+async function updateSheetWithGeocodes(country) {
+  const client = await getAuthorizedClient();
+  let data = {}
+
+  // Open relevant sheet
+  const sheets = google.sheets('v4');
+  const request = {
+    spreadsheetId: SHEETS[country],
+    range: 'Form Responses 1'
+  };
+  request.auth = client;
+
+  // Transition code as we rename the output sheet from
+  // Form Responses 1 to Combined.
+  let response = null;
+  try {
+    response = await sheets.spreadsheets.values.get(request);
+  } catch (err) {
+    request.range = 'Combined';
+    response = await sheets.spreadsheets.values.get(request);
+  }
+  // Find rows that have been approved but not geocoded.
+  // Call geocoder.
+  // Fill in cells with lat, lng
+  await annotateGeocode(response.data, SHEETS[country], client);
+}
+
+module.exports.geocode = functions.https.onRequest(async (req, res) => {
+  const country = req.path.split('/',2)[1] || 'us';
```

Comment:
Can we pull this out into a function:

get_country_from_path(req.path)

so it's not copy/pasted on 2 exports

Reply:
Done.